test/lib.bash

```diff
@@ -358,12 +324,18 @@ function teardown {
 
 # == State dumps
 
-function dump_state {
+function dump_state.setup {
```

Comment:
Why are you using this "dot" style? We use underscore in most places and here it's even combined. I'm missing the purpose.

Reply:
I use dot to indicate a function grouping inside "psudo" object. The same as `logger.info`, `logger.debug`. Then you know that some functions are related to each other.

This is just a preference, though. I'm not strongly pursuing this style.